### DISCUSSION TITLE
Why not a Laser pointeur ?

### DISCUSSION THREAD

**Discussion by biglooping** [geovens/gInk#207] [Ideas]
Hello,
I sometimes use this application and I'm very grateful for it.

When sharing a screen with friends, I sometimes want to show a part of the screen but I don't want to write a line, a shape or anything else.

This can be done with a new tool: a laser pointer.    

Like a simple yellow cercle  ,  50 % transparency ,  radius' size adjustable (5 to 50 pixels).

Is it possible to have such a feature in a future version?

And do you think it could be useful to anyone?

Thanks in advance 

Biglooping.

**Comment by pubpub-zz** [comment]
You have the spot mode : press Ctrl to activate it or select it (within the zoom button in the toolbar)
Ps this feature is available in my fork https://github.com/pubpub-zz/ppInk

**Reply by biglooping** [reply]
Hello  pubpub-zz,

I read the GIT page project, thanks for your work done on the ppInk application.

I tested  your application but no "lazer pointer" as it is used in one of your tutorial video  "02-38-18.mp4".

I would like to have such yellow circle , can i do it with the application ? 

Or should I customize the mouse pointer ?

Best regards

biglooping

**Reply by pubpub-zz** [reply]
oups: it is not ctrl but alt key.
the spot mode currently draws a gray area outside of the spot circle. outside color (in RGBA) and radius can be adjusted directly in the config.ini
If you really want a yellow circle only you should draw your own a new cursorarrow. This cursor is used when spot mode is active. It is also used with some tools (eg text insertion, moving objects...) but this should be achievable
